pkg/admission/apiexport/admission.go

```diff
@@ -99,5 +100,25 @@ func (e *APIExportAdmission) Validate(ctx context.Context, a admission.Attribute
 		}
 	}
 
+	for i, rs := range ae.Spec.ResourceSchemas {
+		if err := validateResourceSchema(rs, field.NewPath("spec").Child("resourceSchemas").Index(i)); err != nil {
+			return admission.NewForbidden(a, err)
+		}
+	}
+
+	return nil
+}
+
+func validateResourceSchema(resourceSchema apisv1alpha2.ResourceSchema, path *field.Path) *field.Error {
+	group := resourceSchema.Group
+	if group == "" {
+		group = "core"
```

Comment:
does this mean group should have a `""` default? Is it documented in the API docs that empty group means core?

Reply:
Added as doc line in the api